app/components/failed_network_action/index.js

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {TouchableOpacity, View} from 'react-native';
+
+import FormattedText from 'app/components/formatted_text';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import Cloud from './cloud';
+
+export default class NoResults extends PureComponent {
```

Comment:
I think it's preferable to name this like `FailedNetworkAction`.

Reply:
oh yes my bad, I'll fix this